pkg/server/team/team.go

```diff
@@ -95,9 +90,25 @@ func (dbt *DatabaseOperations) ListByUser(userEmail string) ([]*storage.Team, er
 			errors.Wrap(err, fmt.Sprintf("finding teams of user %s", userEmail)),
 		)
 	}
+
+	if err = dbt.associateUsersWithTeams(teams); err != nil {
+		return nil, err
+	}
 	return teams, nil
 }
 
+func (dbt *DatabaseOperations) associateUsersWithTeams(teams []*storage.Team) error {
```

Comment:
I don't like this function name. It associates users with teams and perform a find operation, so it seems it goes a _little_ beyond the association.